content/components/selectpanel.mdx

```diff
@@ -143,30 +110,76 @@ When no results are available, it is essential to offer users an explanation ins
-4. **Confirmation**: If your layout does not incorporate optimistic updates or lacks error messaging for failed updates, it is crucial to use a confirmation button as a loading state to clearly communicate the progress of the update.
+1. **Initial**: Used during initial data retrieval to avoid displaying an empty list.
+2. **Initial Skeleton**: Used similarly for initial data retrieval, presenting a skeleton screen instead of an empty list.
+3. **Searching/Filtering**: A minimal loading indicator is used during searches, allowing users to retain or continue their selections.
+4. **Confirmation**: Essential when updates aren't instant or when there's no error messaging for failures, using a confirmation button to indicate update progress.
+
+## Behavior
+
+### Opening
+
+<img
+  width="960"
+  alt="Various trigger examples are displayed. The first example showcases an icon button with a cog icon. The second example presents a button with the internal label 'Assignees: 2 people'. The third example features a button with the external label 'Assignees' and an internal button label stating '2 people'. Lastly, the image shows an action menu with two items: 'Set labels...' and 'Delete...'."
+  src="https://github.com/primer/design/assets/980622/f2867468-4606-4999-8ea7-21860e50051d"
+/>
+
+Select panels can be activated by regular [buttons](/components/button), [icon buttos](/components/icon-button), or [action menu items](/components/action-menu). Proper labeling is crucial for representing current selections, either internally (within the button) or externally (adjacent to the button). Focus will be placed on the first interactive element within the select panel.
+
+<Note>
+
+  When the select panel is activated through an [action menu](/components/action-menu), it should be displayed as a centered dialog using the `dialog` property.
+
+</Note>
+
+### Closing
+
+To close the select panel, pressing 'Esc', selecting a 'Cancel' option or clicking outside will close the dialog, and return focus to the trigger element without retaining or submitting any user input. Conversely, clicking 'Submit' or pressing 'Enter' while focused in the listbox will submit the user's input.
+
+### Appearance
+
+<img
+  width="1338"
+  alt="Two examples of multi-select panels. The first panel is anchored to a Set labels button. When the button is clicked, the multi-select panel opens adjacent to the button, allowing the user to make selections. The second panel is displayed as a centered dialog. When triggered, it appears at the center of the screen, allowing users to interact with the multi-select options in a modal-like interface."
+  src="https://github.com/primer/design/assets/980622/9fd418af-5050-423a-af8e-93b30571044a"
+/>
+
+The select panel can appear as either an anchored or centered dialog, depending on the trigger source.
```

Comment:
I don't know why this was removed, it's helpful information especially since we mention that it's dependent on the trigger source. 

```suggestion
The select panel can appear as either an anchored or centered dialog, depending on the trigger source.

- **Anchored dialog**: When triggered by a [button](/components/button) or [icon button](/components/icon-button) but never by an [action menu](/components/action-menu).
- **Centered dialog**: When triggered by a [button](/components/button), [icon button](/components/icon-button) or [action menu](/components/action-menu).

On smaller breakpoints the select panel should always appear as a modal dialog.
```